frontend/src/components/EntityBulkAdd.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div v-for="(form, index) in entityForms" :key="form.props.id">
+    <div :id="`form-${index}`">
+      <div :class="`${entityForms.length < 2 && 'd-none'} ${index > 0 && 'mt-5 border-top border-light'} row`">
+        <div class="col">
+          <button type="button" :class="`btn-close float-end ${index > 0 ? 'my-3': 'mb-3'}`"
+                  @click="closeForm(form.props.id)"/>
+        </div>
+      </div>
+      <component :is="form.component" @save-all="saveAll" v-bind="form.props"
+                 :ref="el => { refs[`entity-form-${index}`] = el }"/>
+    </div>
+  </div>
+  <div class="container mt-2">
+    <button class="btn btn-outline-secondary w-100" @click="addNewItem">
+      <i class='eos-icons'>add_circle</i>
+      Add more
+    </button>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'EntityBulkAdd'
+}
+</script>
+
+<script setup>
+import {markRaw, ref} from "vue";
+import EntityForm from "@/components/inputs/EntityForm.vue";
+
+const props = defineProps({
+  schema: {
+    type: Object,
+    required: true
+  },
+  attributes: {
+    type: Object,
+    required: true,
+  }
+});
+
+const entityForms = ref([generateEntityForm(0)]);
+const refs = ref([]);
+
+function generateEntityForm(id) {
+  return {
+    component: markRaw(EntityForm),
+    props: {
+      id: `entity-form-${id}`,
+      attributes: props.attributes,
+      schema: props.schema,
+      batchMode: true,
+    }
+  }
+}
+
+async function saveAll() {
+  let successIds = [];
+  const promises = Object.entries(refs.value).map(async x => {
+    const refName = x[0];
+    const component = x[1];
+
+    if (refName && component) {
+      const resp = await component.createEntity();
+      if (resp?.id) {
+        successIds.push(refName);
+        component.editEntity.name = null;
+        component.editEntity.slug = null;
+      }
+    }
```

Comment:
Should we handle the case of this condition being false? E.g. via a warning in the JS console? Or I'm I too cautious here?

Reply:
I think we don't need it. I'm collecting only components which have successful response in order to exclude them later. The ones, which caused errors will still remain displayed and a message will pop up to the user.